modules/vaos/app/services/vaos/v2/appointments_service.rb

```diff
@@ -81,6 +82,11 @@ def post_appointment(request_object_body)
                        perform(:post, appointments_base_path_vaos, params, headers)
                      end
 
+          if request_object_body[:kind] == 'clinic' &&
```

Comment:
@cferris32 The only difference between this PR and what we worked on during our pair session is that I moved this date modification block from the top of the method to after the post requests. 

I noticed while fixing tests tests that in cases where patientIcn is missing, we would make a redundant get facilities request if we had this logic at the beginning of this method. Instead, if we had this logic here, the post requests above will raise exceptions first and we won't make the get facilities request which I think is slightly better. Let me know if you disagree and I can move the logic back to the top of the method and update the VCR cassettes instead.

Reply:
I think it should be good this way, that's a nice catch and saving on a redundant call is preferred. I don't think there should be anything else this impacts